packages/ckeditor5-dev-webpack-plugin/package.json

```diff
@@ -6,7 +6,12 @@
   "main": "lib/index.js",
   "dependencies": {
     "@ckeditor/ckeditor5-dev-utils": "^9.0.1",
-    "rimraf": "^2.6.2"
+    "chalk": "^2.4.1",
```

Comment:
Is it imported anywhere?

Reply:
https://github.com/ckeditor/ckeditor5-dev/pull/373/files/127cd77aa4d7d048b125896ef8448ce31443ad63#diff-088f1ffe6ad43d8c12490e6e14c9c73bR8